### PR TITLE
Editor settings UI tooltips

### PR DESCRIPTION
**Pull request by ernopp**
Update Editor & Runtime settings UI tooltips based on Valentyn and Anne's suggestions here

- [Editor settings](https://docs.google.com/document/d/1Q52o6YT-B9Bm1JkfX4JhTF0XGoXCN2Lu-98LhzQIC0M/edit)
- [Runtime settings](https://docs.google.com/document/d/1W2qxeW-mUxRXlHT5Xpy5TVYX2C4C_o-SP0vGuHtaJrE/edit)

We want to document as best we can the settings in context (ie as tooltips) rather than have to keep a docs page in sync.

**Testing**
This PR only changes display values. 
I have only ensured that I could rebuild the GDK after the changes, which succeeded.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h [hunk 1]
```diff
@@ -23,40 +23,60 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
...
 	float HeartbeatTimeoutSeconds;
 
 	/**
-	 * Limit the number of actors which are replicated per tick to the number specified.
-	 * This acts as a hard limit to the number of actors per frame but nothing else. It's recommended to set this value to around 100~ (experimentation recommended).
-	 * If set to 0, SpatialOS will replicate every actor per frame (unbounded) and so large worlds will experience slowdown server-side and client-side.
-	 * Use `stat SpatialNet` in editor builds to find the number of calls to 'ReplicateActor' and use this to inform the rate limit setting.
+	 * Specifies the maximum number of Actors replicated per tick.
+	 * Default: `0` per tick  (no limit)
+	 * To avoid game client and server-worker instance slowdown, set the value to around `100`
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h]: I'd remove this line. We're seeing internally that we want it much higher, or off entirely.

File: SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h [hunk 2]
```diff
@@ -92,7 +112,11 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Metrics", meta = (ConfigRestartRequired = false), DisplayName = "Metrics Report Rate (seconds)")
 	float MetricsReportRate;
 
-	/** Change 'Load' value in inspector to represent worker Frame Time instead of a fraction of target FPS.*/
+	/** 
+	* By default the SpatialOS Runtime reports server-worker instance’s load in frames per second (FPS). 
+	* Select this to switch so it reports as seconds per frame.
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h]: Is this not milliseconds per frame?

Comment (ernopp) [reply SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h]: according to this doc's description it's in seconds per frame

https://docs.google.com/document/d/1W2qxeW-mUxRXlHT5Xpy5TVYX2C4C_o-SP0vGuHtaJrE/edit

Comment (ernopp) [reply SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h]: Im going to confirm

Comment (ernopp) [reply SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h]: Confirmed in inspector

**Pull request merged**